libs/ngx-dsfr/radio/src/lib/radio.component.ts

```diff
@@ -61,38 +70,50 @@ export class DsfrRadioComponent implements ControlValueAccessor, OnInit {
     ngControl.valueAccessor = this;
   }
 
+  ngOnChanges (changes: SimpleChanges): void {
+    if (this.hasFailed && (!this.failureId || !this.failureMessage)) {
+      throw EMPTY_FAILURE_ID_OR_MESSAGE_ERROR;
+    }
+    if (this.hasSucceeded && (!this.successId || !this.successMessage)) {
```

Comment:
Don't forget to change these 2 `if` conditions with the removal of failureId and successId